src/components/Badge.jsx

```diff
@@ -1,28 +1,20 @@
 import styled from "styled-components"
 
-export default function Badge({ type, size = "normal" }) {
-  if (type === "new") {
-    return (
-      <StyledBadge type={type} size={size}>
-        NEW
-      </StyledBadge>
-    )
-  } else if (type === "best") {
-    return (
-      <StyledBadge type={type} size={size}>
-        BEST
-      </StyledBadge>
-    )
-  }
+export default function Badge({ message }) {
+  let type = ""
+  if (message === "NEW") type = "new"
+  else type = "best"
+
+  return <StyledBadge type={type}>{message}</StyledBadge>
```

Comment:
message를 소문자로 받고(e.g. new) 사용할 때 대문자 변환시키는건 어떨까요?
```suggestion
  return <StyledBadge type={type}>{message.toUpperCase()}</StyledBadge>
```